readme.md

```diff
@@ -480,10 +480,10 @@ XO is based on ESLint. This project started out as just a shareable ESLint confi
 
 ## Badge
 
-Show the world you're using XO → [![XO code style](https://img.shields.io/badge/code_style-XO-5ed9c7.svg)](https://github.com/xojs/xo)
+Show the world you're using XO → [![XO code style](https://shields.io/badge/code_style-5ed9c7?logo=xo&labelColor=gray)](https://github.com/xojs/xo)
 
 ```md
-[![XO code style](https://img.shields.io/badge/code_style-XO-5ed9c7.svg)](https://github.com/xojs/xo)
+[![XO code style](https://shields.io/badge/code_style-5ed9c7?logo=xo&labelColor=gray)](https://github.com/xojs/xo)
 ```
```

Comment:
Maybe we should also link to your issue with examples of some other styles? I don't mind if user customise it to their liking.

Reply:
Sure. I added one hint below this line.

> Or [customize your badge as you want](https://github.com/xojs/xo/issues/689#issuecomment-1253127616).